recipes/dunovo/meta.yaml

```diff
@@ -0,0 +1,41 @@
+
+package:
+  name: dunovo
+  version: 0.6
+
+source:
+  fn: v0.6.tar.gz
+  url: https://github.com/galaxyproject/dunovo/archive/v0.6.tar.gz
+  sha256: a3cc378c22fcfc3bcaa11129f27f61bab5ff51cc9185320d53ece273b56e6214
+
+build:
+  number: 0
+
+requirements:
+  skip: True  # [py3k or osx]
+  build:
+    - make
+    - gcc
+
+  run:
+    - libgcc
+    - python >=2.6,<3
+    - argparse  # [py26]
```

Comment:
this means that argparse isn't a dependency if python version != 2.6, is this intended ?
keep in mind that since python3 was already skipped in the `skip:True` line, so you don't have to worry about it in the requirements, `python<3` is redundant.